shared/src/proxies/logErrorProxy.ts

```diff
@@ -0,0 +1,15 @@
+import { ClientApplicationContext } from '@web-client/applicationContext';
+import { post } from './requests';
+
+export const logErrorInteractor = (
```

Comment:
The ticket specifies that they still want the errors logged to Opensearch.

Reply:
TODO: Add authorization check so arbitrary users cannot log errors by spamming the endpoint